config/tsconfig.emit-declarations.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.workspace.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "noEmit": false
+  }
```

worth adding our `outDir` and `rootDir` here?

```suggestion
  "compilerOptions": {
    "outDir": "../types",
    "rootDir": "."
    "declaration": true,
    "declarationMap": true,
    "emitDeclarationOnly": true,
    "noEmit": false
  }
```

so simplify our packages extending it

---

unsure how relative paths are determined. it might be from the extended config or from the extendee config.  will give it a shot.

---

yeah, it works!

---

Well.  OK, so .. it works for _some_ properties.  It works for `outDir`.  It doesn't work for `rootDir`.  It doesn't work for `references`, either.  

`rootDir` only needs setting if it isn't `.`, anyway.

---

No, it doesn't work.  I had confused myself somehow.  All paths are relative to the path of the config file.